transaction/pane.js

```diff
@@ -198,15 +197,15 @@ module.exports = {
         var c3 = tr.appendChild(dom.createElement('td'))
         var amount = kb.any(x, ns.qu('in_USD'))
         c3.textContent = amount ? d2(amount.value) : '???'
-        c3.setAttribute('style', 'width: 6em; text-align: right; '); // @@ decimal alignment?
+        c3.setAttribute('style', 'width: 6em; text-align: right; ') // @@ decimal alignment?
```

Comment:
Perhaps a CSS class?

Reply:
Using classes to trigger CSS style is document-wide.  You can't inject style sheets i a subtree.  So CSS classes  is a bit of  global file space, and so I avoid it.  You can't really use it with libraries.